### DISCUSSION TITLE
Removal of Top 10 Masteries (High Scores)

### DISCUSSION THREAD

**Discussion by BahamutVoid** [RetroAchievements/RAWeb#1388] [Ideas]
Revisions are a necessary and healthy part of the site that frequently receive backlash; it's becoming a moderation nightmare to deal with. Since I've been on the site from 2020 I have seen, on both the site and in the discord, the demonisation of developers that revise popular sets. While we could crack down on people who aren't respecting the developer or the revision process we're going through, that's impractical and would require active moderator supervision for every single revision released. Instead, I think it's better to deal with the root cause. Currently a revision does not remove a mastery badge; it only affects the High Scores section as previous masters are knocked off the list.

What a focus on **Top 10 Masteries** affects:
- Developers feeling discouraged or even outright dejected about their own work.
- Belief that revisions are not healthy for the site, and that new achievements should be relegated to subsets only.
- Arbitrarily limits the showcase to 10 people, which isn't compatible with how much the site has grown.
- Lack of mistrigger tickets, because exploits allow for faster masteries.
- Developers can be criticised for playing their own sets, as this effectively "steals" a Mastery slot due to their set knowledge.
- Developers may be pressured to pad their revisions so that it justifies the revision, or instead omit any challenges to avoid criticism. Both extremes give unnecessary pressure for the revision to be received well.

Ultimately I believe leaving the **Latest Masters** section would be enough, as this instead highlights how many masteries the set has and how recent they are. A developer shouldn't be put in a position where they feel the need to defend a revision that has already been voted on and approved.

Another option would be instead to keep High Scores but change it to exclude those with masteries. This instead shows people who are closest to a mastery. A benefit of this is that it preserves the mechanic of what the section looks like before a set is mastered.

**Comment by redwizard42** [comment]
I personally like seeing top masteries.

That said, these are good points.  I also had the idea that it'd be nice to capture the top 10 for each set version. So there was a record. When I revised DQ8, I preserved the original top 10 list on game wall and forum. I'm sure doing this automatically is far from trivial, however.

**Comment by Soulsderfuchs** [comment]
Top 10s are a good system to have, removing them would not fix the bad reputation revisions have. Seems the focus should be more on informing and educating the players on what a revision is instead of removing features people use and like (me included).

**Comment by AuburnRDM** [comment]
If anything, I'd like to see it not be limited to the Top 10 point earners on the game. The snapshot window showing 10 is fine, but it would be nice to be able to expand the list and see where every user who played the set stands. Currently, the only easy way to see how many points someone has on a set is if they're someone you follow or if you specifically type their name into the compare field.

**Comment by CarnavalSines** [comment]
While I don't mind either way (Top 10 or no Top 10), I agree with Souls that the players complaining about revisions using it as an argument probably don't really care about it in the first place and would just find something else if it was removed.

**Comment by FRAN269** [comment]
I strongly disagree with Top 10 Masteries being the issue in relation to the belief that revisions are not healthy for the site, removing it would not change that at all, I think if anything this feature could improve, allowing us to see further than just the Top 10 as Auburn mentioned, removing it fixes nothing, the actual root cause of this is people being angry that they have to replay and remaster a game they already went through the effort of mastering before, and that will never change, people will always be annoyed by that, removing the Top 10 Masteries would not make them any less annoyed.

**Comment by Maximdraco** [comment]
I feel like the First 10 masteries is just a extension of the already existing bad reaction to revisions for hard/relative long games, I personally don't care if the First 10 masteries dissapear or no (I would preffer just a list of masteries than could be sorted by date), if it dissapear it will cause negative reactions.

I think users should simply get used to revisions, I don't think is a problem on our side (website and devs), is a problem on they side (users who doesn't like revisions) for the way they see them.

**Comment by ladynadiad** [comment]
https://retroachievements.org/viewtopic.php?t=681&c=160331#160331

This response shows that the argument of they'll just find something else to complain about if we take away the top 10 isn't a solid reason to fail to consider this suggestion.  This user clearly doesn't object to this sort of revision other than the fact that it takes away the top 10.

And it's definitely a valid point that the top 10 masteries just doesn't have as much reason to exist as it did when the site was first created.  Back then sets were commonly released incomplete and there were fewer users.  Once the top 10 is filled, it's useless beyond showing that the set is able to be mastered.

As things stand right now top 10 masteries is used as a justification to many behaviors that are hurtful to devs when it really isn't an important stat to anyone besides the user who has it and to show that the set is doable to master.  I'd much rather see it replaced with a way to see all users who have mastered the set, not just the first and most recent 10.

**Reply by AuburnRDM** [reply]
I wouldn't mind if the data was expanded upon in a different way. You could have the list of the "Closest to mastery" like BV suggested as well as the latest masteries, but those should expand to another page where you can see _all_ of the mastery data. This page would still list every mastery in order (as well as other sorting options), but there would no longer be a "Top 10" attached to it, and overall devalue that part of it. The data should be there for those who want to dig into the stats, but it doesn't need to be on the game page itself.

**Comment by Fridge-Box** [comment]
I agree with the proposal. The "Top 10" doesn't hold value when it can be completely wiped by a 1 achievement revision.
 Expanding the list of historical data would also be fine, like if a person mastered a set in 2016, let them keep their #1 spot. A revision shouldn't nullify that spot. People will compete over the #1 spot of a mastery list. We don't reset set Leaderboards when a revision happens, like for high scores, so why would another metric be reset?

**Comment by Jamiras** [comment]
#2583 eliminated the "Top 10 masteries". The widget now shows the "Latest Masteries", with a link to browse all masteries.

The list is still affected by revisions as users will no longer be considered as having mastered the set. That's better tracked by #1240.
